src/main/scala/trace/TacitEncoder.scala

```diff
@@ -0,0 +1,408 @@
+// See LICENSE.Berkeley for license details.
+// See LICENSE.SiFive for license details.
+
+package freechips.rocketchip.trace
+
+import chisel3._
+import chisel3.util._
+import scala.math.min
+
+import org.chipsalliance.cde.config.Parameters
```

Comment:
This should be removed from RC then? The abstract encoder is in RC, but the custom encoder should be elsewhere?